src/spikeinterface/sortingcomponents/benchmark/benchmark_matching.py

```diff
@@ -149,9 +536,10 @@ def plot_errors_matching(benchmark, unit_id, nb_spikes=200, metric="cosine"):
         ax.set_xlabel(metric)
 
         count += 1
+    return fig, axs
 
 
-def plot_errors_matching_all_neurons(benchmark, nb_spikes=200, metric="cosine"):
+def plot_errors_matching_all_neurons(benchmark, comp, nb_spikes=200, metric="cosine"):
```

Comment:
Here we break a bit the previous behavior : a benchmark was self contaon andready for ploting.
Here the comp object must externaly given.
I guess we can refactor later.

Reply:
Yeah, but the previous behavior required the benchmark be associated with a particular comparison object, which limits what you can do with a given benchmark object.  I think it's better to have higher-level benchmarks that see multiple methods/comparisons to limit the duplicated work (generating waveform extractors, template similarity, etc.)

Reply:
This is a general problem acrross all benchmar class.
When exploring a grid parameters : some parts are fixed and some parts need recompute.
We should find a general way.
Lets do this later.